app/views/katello/api/v2/subscription_facet/subscription_status.json.rabl

```diff
@@ -0,0 +1,3 @@
+
+attributes :to_global => :color_code
```

Comment:
I would think we'd have values here that make sense to an API user, and in some ways CLI. And that the UI would have code to interpret to a color value. We could stick to a known understanding that the status is represented by a series of numbers that are documented as having known values or use something more readable such as I see in the status model (e.g. error, ok, warn). I just want to make sure the API data is reasonable for the variety of purposes and I don't see any mention of color in https://github.com/Katello/katello/blob/master/app/models/katello/subscription_status.rb for example.